queue_listeners/lib/actionkit_page_parser.rb

```diff
@@ -0,0 +1,61 @@
+require_relative '../config/load_env'
+require_relative '../lib/crm_page'
+require 'json'
+
+class ActionkitPageParser
+  def initialize
+    @base_url = ENV['ACTIONKIT_BASE_URL']
+    @widget_to_page_types = {petition: 'Petition', donation: 'Donation'}
+  end
+
+  def parse_from_message(message)
```

Comment:
I'm not sure we want to do that - I think there's a likelihood we want to parse multiple pages in a row (for instance, we get a response from ActionKit and there are four pages we want to parse sequentially) and in my mind, it doesn't make sense to instantiate a brand new `Parser` object for each of those.

Thoughts?